src/main/java/com/example/kinoxpback/model/Theater.java

```diff
@@ -0,0 +1,64 @@
+package com.example.kinoxpback.model;
+
+import com.example.kinoxpback.enums.TheaterSize;
+import jakarta.persistence.*;
+
+@Entity
+public class Theater {
+    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name="theater_id")
+    private int theaterID;
+    private TheaterSize size;
```

Comment:
Please insert @Collum(name = "") and @Enumerated(EnumType.STRING) for the TheaterSize enum